src/ui/components/VPNToggle.qml

```diff
@@ -27,6 +27,8 @@ VPNButtonBase {
         return VPN.activate();
     }
 
+    onClicked: handleClick()
```

Comment:
This is a problem. I removed it because `handleClick` is executed twice when the user clicks on the button.

Reply:
Can you please check if this happens with your patch?

Reply:
Just tried logging inside `handleClick()` and getting only 1 log message in app output. I believe it looks good.. Is the consequence of double `handleClick` is that the toggle bounces off-on-off? If so it definitely does not happen.